src/screens/AddSiteScreen.js

```diff
@@ -13,6 +13,7 @@ import {
     updateSiteNearestTownText,
     updateAccessibilityOption,
```

Comment:
Below is the `AddSiteScreen` changes I was referring to before where we can now select `price`.